docs/site/index.html

```diff
@@ -8,7 +8,6 @@
       content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1"
     >
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,400italic,500,700" rel="stylesheet" type="text/css">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Yeah, I couldn't decide whether to leave it in anticipation of `FontIcon`, or be clean about it and remove it until it's needed.